packages/react-native-sdk/package.json

```diff
@@ -31,12 +31,16 @@
     "kinvey-js-sdk": "^4.2.3"
   },
   "peerDependencies": {
-    "react-native": "^0.59.9"
+    "@react-native-community/async-storage": "^1.6.1",
+    "react-native": "^0.60.0"
   },
   "devDependencies": {
+    "@react-native-community/async-storage": "^1.6.1",
     "@types/react-native": "^0.60.3",
+    "@types/react-native-sqlite-storage": "^3.3.1",
     "del-cli": "2.0.0",
     "fs-extra": "~8.1.0",
+    "react-native-sqlite-storage": "^3.3.11",
     "typescript": "~3.5.3"
```

Comment:
If we are not using SQLite, we should remove these SQLite dependencies.

Reply:
Yep I agree. I noticed it after I created the PR so I removed SQLite as a dependency with cb411fb.